### DISCUSSION TITLE
Mining Pool

### DISCUSSION THREAD

**Discussion by N1k-92** [BitMaker-hub/NerdMiner_v2#383] [Q&A] [answered]
Hi, do i have to use a solo mining pool or can i just do it without a pool? Dont want to pay fees.

**Comment by MyOwn2C** [comment, marked as answer]
There are 0% fee solo pools for Nerdminers. 
Vkbit and Public-pool.io for example. 
Nerdminer is too slow to join any non-solo pools.
Also, chances of Nerdminer finding a block, and you need to pay fee for that, is astronomically low.

**Reply by IAmSixNine** [reply]
FYI, a few weeks ago vkbit had a site update and my nerdminers no longer connect to it. currently the only ones i know that work are public-pool.io and pools.nerdminers.org
I was hoping to find a BCH pool but i never did find any. 
Also ESP32 devices with larger displays dont show worker info if you connect to anything OTHER than public-pools.io, the info at the bottom displays zeros. So best difficulty, workers and total hash rate dont populate on the screen. But they are still working properly, just not relaying the info. I believe they are coded to look for the json info from public-pools.io
